.pre-commit-config.yaml

```diff
@@ -1,4 +1,7 @@
 ---
+ci:
+    skip: [cargo-clippy, cargo-check, cargo-fmt]
```

Comment:
I understand why it cannot work there, but then, is the idea to the [current](https://github.com/arighi/virtme-ng/blob/main/.github/workflows/rust.yml) Rust GH Action to run `pre-commit` (only the Rust related ones?) to be sure the same commands are executed?

BTW: thank you for adding that! :)